### ISSUE TITLE
static properties don’t show static in their declarations

### ISSUE THREAD

**Issue by prachigauriar** [SwiftDocOrg/swift-doc#217]
I noticed that in swift-doc 1.0.0-beta.5, static properties don’t show the keyword `static` in their declaration. This can be confusing when reading the documentation, as it they look like normal instance properties.

Example Input:

```swift
public struct MyThing {
    public static let staticProperty: String = ""
    public let instanceProperty: Int


    public init(instanceProperty: Int) {
        self.instanceProperty = instanceProperty
    }
}
```

Command-line: 

```sh
swift-doc generate --module-name SwiftDocExample -o Docs
```

Output:

# MyThing

``` swift
public struct MyThing
```

### `init(instanceProperty:)`

``` swift
public init(instanceProperty: Int)
```

### `staticProperty`

``` swift
let staticProperty: String = ""
```

### `instanceProperty`

``` swift
let instanceProperty: Int
```

Please consider fixing this bug for the next beta. Differentiating between static and instance properties is pretty important.

**Comment by Lukas-Stuehrk** [comment]
This problem seems to be fixed with the changes introduced in #204.

@prachigauriar 
This should be fixed already, but there hasn't been a new release yet. You can check the fix if you compile swift-doc from source using the instructions in the README.

**Comment by prachigauriar** [comment]
@Lukas-Stuehrk Great, thanks for letting me know! I’ve installed from `master` and see acceptable behavior. I’ll close the issue.
